.travis.yml

```diff
@@ -17,9 +17,9 @@ addons:
 
 env:
   global:
-    - RUST_COMPILER_VERSION=1.26.2
-    - RUST_NIGHTLY_VERSION=nightly-2018-06-29
-    - RUST_CLIPPY_VERSION=0.0.211
+    - RUST_COMPILER_VERSION=1.28.0
```

Comment:
(here and elsewhere) Shall we still specify a particular version (1.28.0 instead of `stable`)?

Reply:
Agreed, we should specify `stable`